packages/core/src/components/popover/popover.tsx

```diff
@@ -514,9 +521,16 @@ export class Popover extends AbstractComponent<IPopoverProps, IPopoverState> {
             // so instead, we'll position tether based off of its first child.
             // NOTE: use findDOMNode(this) directly because this.targetElement may not exist yet
             const target = findDOMNode(this).childNodes[0];
+            let tetherUtilOptions = this.props.tetherOptions || {};
```

Comment:
can you supply this default with `defaultProps` instead? then you don't need any `tetherUtilOptions` variable. also, we generally prefer to avoid implicit type coercion (here, `this.props.tetherOptions` is coerced to a boolean)